opm/input/eclipse/share/keywords/900_OPM/T/THCO2MIX

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "THCO2MIX",
+  "sections": [
+    "PROPS"
+  ],
+  "size": 1,
+  "items": [
+    {
+        "name": "MIXING_MODEL_BRINE",
+        "value_type": "INT",
+        "default": 3
+    },
+    {
+        "name": "MIXING_MODEL_GAS",
+        "value_type": "INT",
+        "default": 0
+    }
```

Comment:
Ideally I'd like these to be strings instead, since that's more self-explanatory in the input.

Reply:
I see your point. Then I think we need to split it into 3
--MIXING_MODEL_SALT MIXING_MODEL_LIQUID MIXING_MODEL_GAS
THCO2MIX
 NONE IDEAL IDEAL 

will ignore effect of salt (NONE) and assume ideal mixing for CO2+WATER in LIQUID and GAS.